source/feature/openvswitch/native-openvswitch.html.md

```diff
@@ -104,31 +119,34 @@ Minimal changes in current API verbs are required:
 - oVirt currently differentiates between VM (bridged) networks and non-VM networks due to historical performance benefit of non-bridged networks for non-VM payloads (such as storage and migration). This differentiation is moot with OVS, but in order to keep Engine oblivious to the change, VDSM should lie to it, and report bridged=False for networks that requested it. Dropping the differentiation from Engine should happen in a later stage.
 
 ### Upgrade from native to OVS networking
-Transition between switch types must be supported for the host as a whole. Nevertheless, if the user has defined his own networking configuration which is not controlled by VDSM, it should coexist with VDSM controlled networks.
-
-Mixing between native and OVS controlled networks is not initially supported, therefore, when a network is marked with a different type from the existing, a validation check should make sure that all other networks are marked for the same network type.
-
-The following sequence describes the transition steps @Engine:
-
-- Vdsm on hosts is upgraded to an OVS-supporting version (4.0). No change takes place to preexisting network configuration, which would still be bridge-based.
-- When all active hosts in the cluster have been upgraded, the user should upgrade the cluster to version 4.0 where OVS is supported.
-- The default switch type for the cluster should be set by the user to OVS (for new hosts).
-  - Newly-created 4.0 clusters should have OVS as the default switch.
-- Update individual hosts to transition to OVS: 
-  - The host networking can be changed when no VM/s are connected to it. With this option oVirt must support VM migration between different switch types.
-  - We can further improve the user experience by a second approach:
-The host networking can be changed in-place without the need to migrate the VM/s. With this option, migration between hosts that have different networking types is not needed (but can coexist).
-Negligible service interruption to the VM networking is expected.
-
-The following sequence describes the transition steps @Host:
-- Receive a setupNetworks call.
-- Detect that a network has been marked with a type that is different from the current.
-- Validate that all existing networks exist in the command and that all are marked with the same type.
-  - We would like to perform an atomic upgrade, and not to worry about codependent networks with differing switch types. 
-- Tear down the pre-existing networks and bondings..
-- Create networks.
-
-Note: Engine user may decide to take the reverse transition, from OVS to legacy, as well.
+
+Upgrade is currently not supported. Causing networks to go out-of-sync due to differences between native and OvS see [Limitations](#limitations). Only newly added hosts can be configured with OvS networking. Mixing native and OvS in cluster is not supported. Plase note that the change is blocked only in UI but is still possible via REST api. 
```

Comment:
Upgrade is currently not supported. Causing networks to go out-of-sync due to differences between native and OvS see [Limitations](#limitations).
->
Upgrade from native to OVS networking is currently not supported. An upgrade will cause networks to be out-of-sync due to differences between native and OvS (see Limitations).

Reply:
Done